threedgrut/strategy/gs.py

```diff
@@ -101,6 +101,7 @@ def post_optimizer_step(self, step: int, scene_extent: float, train_dataset, bat
 
         return scene_updated
 
+    @torch.no_grad()
```

Comment:
This is probably okay, but a bit outside the scope of this MR (but we can merge it I guess). But just out of curiosity, do you see any issues with not adding `@torch.no_grad()` here?